CHANGELOG/schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "message": {
+      "type": "string",
+      "description": "Message of the changelog",
+      "minLength": 1,
+      "maxLength": 1000
+    },
+    "type": {
+      "type": "string",
+      "description": "Changelog type",
+      "enum": [
+        "feature",
+        "bugfix",
+        "dependency",
+        "deprecation",
+        "breaking_change"
+      ]
+    },
+    "scope": {
```

Comment:
If you make this a string we may run into issues like:

`plugin` and `plugins` 
Misspelled product names

What if you made this an enum with valid options, and included a `misc` keyword for things that dont fit into the options?

Reply:
Sounds good to me. enum value proposal is welcome. Regarding the `misc`, we can use null to replace it. changelog entry that set the `scope` to null or missing it will be grouped to a default scope.

For example
```
# 1.yaml
message: add a feature
type: feature
pr: 1

# 2.yaml
message: add b feature
type: feature
scope:
pr: 2
```

these two changelog entries will be grouped into a default scope.